pkg/adapter/nimbus-kyverno/processor/kpbuilder.go

```diff
@@ -239,75 +351,313 @@ func cocoRuntimeAddition(np *v1alpha1.NimbusPolicy) ([]kyvernov1.Policy, error)
 	return kps, multierr.Combine(errs...)
 }
 
-func escapeToHost(np *v1alpha1.NimbusPolicy, rule v1alpha1.Rule) kyvernov1.Policy {
+func virtualPatch(np *v1alpha1.NimbusPolicy) ([]kyvernov1.Policy, error) {
+	rule := np.Spec.NimbusRules[0].Rule
+	requiredCVES := rule.Params["cve_list"]
+	//  tbd
+	// schedule := rule.Params["schedule"][0]
+	var kps []kyvernov1.Policy
+	resp, err := utils.FetchVirtualPatchData[[]map[string]any]()
+	if err != nil {
+		return kps, err
+	}
+	for _, currObj := range resp {
+		image := currObj["image"].(string)
+		cves := currObj["cves"].([]any)
+		for _, obj := range cves {
+			cveData := obj.(map[string]any)
+			cve := cveData["cve"].(string)
+			if utils.Contains(requiredCVES, cve) {
+				// create generate kyverno policies which will generate the native virtual patch policies based on the CVE's
+				karmorPolCount := 1
+				kyvPolCount := 1
+				netPolCount := 1
+				virtual_patch := cveData["virtual_patch"].([]any)
```

Comment:
Follow the camelCase naming convention.
```suggestion
				virtualPatch := cveData["virtual_patch"].([]any)
```